LightMessaging/LightMessaging.h

```diff
@@ -1,3 +1,5 @@
+#include <Foundation/Foundation.h>
```

Comment:
I'd prefer to include `CoreFoundation` instead. This would allow `LightMessaging` to continue compiling in `c` (`Foundation` would require `objc`).

```suggestion
#include <CoreFoundation/CoreFoundation.h>
```